scripts/build.py

```diff
@@ -459,6 +459,33 @@ def get_font_filepaths(dependency_bundle, dependency_dir):
     return font_filepaths
 
 
+def add_hashes_to_app_yaml(hashes):
```

Comment:
Can you add a test for this in build_test.py? Validate the final format of the app.yaml file.

Reply:
Removed